ios/ZulipMobile/ZLPConstants.swift

```diff
@@ -0,0 +1,35 @@
+import Foundation
+
+@objc(ZLPConstants)
+class ZLPConstants: NSObject {
+
+
+  // For why we include this, see
+  //   https://reactnative.dev/docs/0.68/native-modules-ios#exporting-constants
+  @objc
+  static func requiresMainQueueSetup() -> Bool {
+    // Initialization may be done on any thread; we don't need access to
+    // UIKit.
+    return false
+  }
+
+  @objc
+  func constantsToExport() -> [String: Any]! {
+    var result: [String: Any] = [:]
+    result["resourceURL"] = Bundle.main.resourceURL!.absoluteString
```

Comment:
Mmm, good question. I started with [an online Objective-C to Swift converter](https://swiftify.com/converter/code/), which, for

```objective-c
  return @{
           @"resourceURL": [[[NSBundle mainBundle] resourceURL] absoluteString]
           };
```

, gave

```swift
return [
    "resourceURL": Bundle.main.resourceURL?.absoluteString ?? ""
]
```

. Then I changed the expression to

```swift
Bundle.main.resourceURL!.absoluteString
```

with the idea that the `!` would cause an error when `Bundle.main.resourceURL` is empty (`nil`, from a look at the [doc](https://developer.apple.com/documentation/foundation/bundle/1410786-main)). That seemed closer to the current behavior.

Reply:
Cool, makes sense.

That doc is also helpful for confirming the translation of APIs. The language switcher at the top takes you between these two:
https://developer.apple.com/documentation/foundation/bundle/1410786-main
https://developer.apple.com/documentation/foundation/nsbundle/1410786-mainbundle?language=objc
so confirming that `Bundle.main` is the translation of `[NSBundle mainBundle]`.